packages/directive-functions-plugin/tests/compiler.test.ts

```diff
@@ -185,12 +210,18 @@ describe('server function compilation', () => {
       const namedFunction_createServerFn_namedFunction = createServerRpc("test_ts--namedFunction_createServerFn_namedFunction", function namedFunction() {
         return 'hello';
       });
+      const namedGeneratorFunction_createServerFn_namedGeneratorFunction = createServerRpc("test_ts--namedGeneratorFunction_createServerFn_namedGeneratorFunction", function* namedGeneratorFunction() {
+        return 'hello';
```

Comment:
For test, it might be good to also `yield` a value before returning

Reply:
Yup makes sense! I'll add this.